config/docker/thinking_sphinx.yml

```diff
@@ -5,15 +5,13 @@ common: &sphinx
   html_remove_elements: "style, script"
   html_index_attrs: "img=alt,title; a=title"
   min_infix_len: 3
-  enable_star: 1
   pid_file: <%= ENV.fetch('THINKING_SPHINX_PID_FILE') { Rails.root.join('log', "searchd.#{Rails.env}.pid") } %>
 
   # This is from TS FAQ - makes reindex MUCH faster.
   sql_range_step: 2000000000
 
-  # # temporarily disabled untill we upgrade sphinx
-  # index_exact_words: true
-  # expand_keywords: true
+  index_exact_words: true
+  expand_keywords: true
```

Comment:
What do these do? will we have partial matches still?

Reply:
After re-reading the definitions I think they are not so useful, really.
- `expand_keywords` adds the whole document (as opposed to infix, prefix which divide the document into smaller searchable pieces) in order to improve search quality. In our app this makes no sense, because we always sort search results (either alphabetically or by last updated/created). [[ref](http://sphinxsearch.com/docs/current/conf-expand-keywords.html)]
- `expand_keywords` doesn't seem to give any advantages. However, it increases indexing time so... if it's been working find until now, I would also remove this one. [[ref](http://sphinxsearch.com/docs/current/conf-index-exact-words.html)]

Reply:
And also, there's this warning 😆 so I guess none of our indexes are compatible with this option.
```
WARNING: no morphology, index_exact_words=1 has no effect, ignoring
```